### PR TITLE
Adds support for more password hashing schemes for converters

### PR DESCRIPTION
**Pull request by VBGAMER45**
Adds support for more password hashing schemes for converters requiring less file edits that need to be made.

Signed-off-by: Jonathan <vbgamer45@gmail.com>

### CODE REVIEW

File: Sources/LogInOut.php [hunk 1]
```diff
@@ -23,7 +23,7 @@
  *  It caches the referring URL in $_SESSION['login_url'].
  *  It is accessed from ?action=login.
  *
- * Uses Login template and language file with the login sub-template.
+ * @uses Login template and language file with the login sub-template.
```
Comment (live627) [inline Sources/LogInOut.php]: Incorrect use of tag; please revert

Comment (VBGAMER45) [reply Sources/LogInOut.php]: Damn how did that get there.

Comment (VBGAMER45) [reply Sources/LogInOut.php]: Reverted should be better now.

Comment (live627) [reply Sources/LogInOut.php]: I  guess you were working on an old file and staged too many changes at once.

Comment (VBGAMER45) [reply Sources/LogInOut.php]: Yeah more than likely I thought just pulled it down but could have been comparing from an older github release with the files on my computer.

**Pull request merged**